test/internal/helpers/Player.java

```diff
@@ -326,6 +327,58 @@ public static Cleanups withClanLoungeItem(final int itemId) {
     return new Cleanups(() -> ClanLoungeRequest.setClanLoungeItem(itemId, 0));
   }
 
+  /**
+   * Restores Clan Furniture after cleanup
+   *
+   * @return Resets to previous state
+   */
+  public static Cleanups withClanFurniture() {
```

Comment:
this function removes added items but doesn't add items that were removed. Also with `withClanFurniture(final String furniture)` cleaning up after itself I'm not sure when this would be used?

Reply:
When you interact with your clan lounge it populates your whole clan rumpus room by reading the entire page. I can just be explicit about each one though. I'll refactor these functions

Reply:
ah, if it's doing that I think it's worth having one to reset everything (like the one that resets fights). I just wasn't clear that was what it was supposed to be doing. `resetLoungeVisit()`?